lib/packBuffers.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var getAccessorByteStride = require('./getAccessorByteStride');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+
+module.exports = packBuffers;
+
+/**
+ * Repacks the accessed buffer data into contiguous chunks.
+ * Also has the effect of un-interleaving interleaved accessors.
```

Comment:
Is it reasonable/useful to un-interleave as a separate stage since several other stages will also need to depend on it?